packages/frontend/src/pages/VerifyEmail.tsx

```diff
@@ -74,41 +57,37 @@ export const VerifyEmailPage: FC = () => {
             <Helmet>
                 <title>Verify your email - Lightdash</title>
             </Helmet>
-            <LogoWrapper>
-                <Logo src={LightdashLogo} alt="lightdash logo" />
-            </LogoWrapper>
-            <FormWrapper>
-                <CardWrapper elevation={2}>
-                    <VerifyEmailForm
-                        email={user.data?.email || 'your e-mail.'}
-                        methods={methods}
-                        data={data}
-                        expirationTime={data?.otp?.expiresAt || new Date()}
-                        onSubmit={({ code }) => {
-                            mutate(code);
-                        }}
-                        onResend={sendVerificationEmail}
-                        isLoading={statusLoading || emailLoading}
-                        verificationLoading={verificationLoading}
-                    />
-                </CardWrapper>
-                <FormFooterCopy>
+            {/* FIXME: use Mantine sizes for width */}
+            <Stack w={400} mt="xl" pt="lg">
+                <Image
+                    src={LightdashLogo}
+                    alt="lightdash logo"
+                    width={130}
```

Comment:
`8xl` I guess? (128px)

Reply:
leaving hardcoded widths for now, added comments to find them easily!